src/main/scala/db/migration/V11__CreatedByNdlaStatusForOwnersWithRoles.scala

```diff
@@ -80,21 +80,49 @@ class V11__CreatedByNdlaStatusForOwnersWithRoles extends BaseJavaMigration with
       })
   }
 
-  def getOwnerIdsWithRoles: Try[List[String]] = {
+  private def getAuth0Response(token: String, page: Int, pageSize: Int) = {
     val url = s"https://$auth0Host/api/v2/users"
+    Try(
+      Http(url)
+        .header("Authorization", s"Bearer $token")
+        .method("GET")
+        .params(
+          "search_engine" -> "v3",
+          "q" -> """app_metadata.roles:"learningpath:write" || app_metadata.roles:"learningpath:admin" || app_metadata.roles:"learningpath:publish"""",
+          "page" -> page.toString,
+          "per_page" -> pageSize.toString
+        )
+        .asString
+    ) match {
+      case Failure(ex) =>
+        println(s"Could not fetch users at page '$page' with page-size '$pageSize' on url '$url'.")
+        Failure(ex)
+      case Success(res) =>
+        println(s"Successfully fetched users at page '$page' with page-size '$pageSize' on url '$url'.")
+        Success(res)
+    }
+  }
 
+  private def getOwnerIdsWithRolesOnPage(token: String,
+                                         page: Int,
+                                         results: List[String] = List.empty): Try[List[String]] = {
+    val pageSize = 3
```

Comment:
Treng kanskje ikkje være 3 som default pagesize?

Reply:
Oi, det var det jeg brukte for å teste pagineringen :man_facepalming: 
Setter den til 50